client/package.json

```diff
@@ -31,7 +31,8 @@
     "eject": "react-scripts eject",
     "lint": "eslint src",
     "lint:fix": "eslint --fix src",
-    "format": "prettier --write ."
+    "format": "prettier --write .",
+    "format:check": "prettier --check ."
```

Comment:
Maybe we could keep the same format as for "lint".
That is to say that "format" checks and "format:write" corrects (write)

Reply:
You are right!